product-service/src/functions/catalogBatchProcess/handler.ts

```diff
@@ -0,0 +1,54 @@
+import { SNS } from 'aws-sdk';
+import { SQSEvent, SQSHandler } from 'aws-lambda';
+import { DynamoDB } from 'aws-sdk';
+import { productsTableName } from '../../constants';
+
+const dynamoDb = new DynamoDB.DocumentClient();
+const sns = new SNS();
+const createProduct = async(productData: any) => {
+  const params: DynamoDB.DocumentClient.PutItemInput = {
+    TableName: productsTableName,
+    Item: {
+      id: productData.id,
+      title: productData.title,
+      description: productData.description,
+      price: productData.price,
+    },
+  };
+  await dynamoDb.put(params).promise();
+}
+
+const catalogBatchProcess: SQSHandler = async (event: SQSEvent) => {
+  console.log({event});
+  try {
+    for (const record of event.Records) {
+      const message = JSON.parse(record.body);
```

Comment:
if the body is not available, it will through an err as it is in a try block
Here is a link from official example from serverless - 
https://github.com/serverless/examples/blob/master/aws-node-typescript-sqs-standard/sqs/receiver.ts

Reply:
Okay